app/models/patient.rb

```diff
@@ -3,11 +3,18 @@ class Patient < ApplicationRecord
 
   GENDERS = %w[male female transgender].freeze
   STATUSES = %w[active dead migrated unresponsive inactive].freeze
+  RISK_PRIORITIES = {
+    HIGHEST: 0,
```

Comment:
Why do we want the risk priorities to be numbers? Can they just be symbols?

Reply:
Having them as numbers makes it easier to sort them. Making them symbols will require specifying custom comparators.